grunttasks/autoprefixer.js

```diff
@@ -5,10 +5,14 @@
 module.exports = function (grunt) {
   'use strict';
 
+  var config = require('../server/config');
+
+  var staticPath = config.get('server.staticPath');
```

Comment:
It'd be nice if the config options we cared about were loaded and attached to the `grunt` object so that each task didn't have to worry about loading the config. What do you think?